### PR TITLE
New Content into old Content

### PR DESCRIPTION
**Pull request by snacksncode**
- Installed `classnames` package
- added proper images to sidebar (+ links)
- all phone numbers and email addresses are now href-ed
- removed unnecessary pages ( gallery )
- fixed some things being rendered wrongly in IE11 (mostly images)
- added PDF's
- moved some styles to global ones (title, table, 3 image container)

### CODE REVIEW

**Comment by Darex1991** [comment]
@snacksncode @SteezyCodes please split this PR into two PRs
Screenshot 2020-06-23 at 01 23 34

File: package.json [hunk 1]
```diff
@@ -10,22 +10,18 @@
     "make-it-pretty": "yarn prettier --config .prettierrc.json --write '{components,pages}/**/*.{jsx,js}'",
     "make-it-stylish": "yarn stylelint '{components,pages}/**/*.scss' --fix --syntax scss --config .stylelintrc.json",
     "make-it-perfect": "yarn run make-it-pretty && yarn run make-it-stylish",
-    "lint": "eslint '{components,pages}/**/*.{jsx,js}' && stylelint '{components,pages}/**/*.scss' --config .stylelintrc.json"
+    "lint": "eslint ./components --fix --ext .js,.jsx"
```
Comment (Darex1991) [inline package.json:13]: please don't change it

Comment (Darex1991) [reply package.json:13]:
```
"lint": "eslint '{components,pages}/**/*.{jsx,js}' && stylelint '{components,pages}/**/*.scss' --config .stylelintrc.json",
    "eslint-components-windows": "eslint ./components --fix --ext .js,.jsx",
    "eslint-pages-windows": "eslint ./pages --fix --ext .js,.jsx",
    "eslint-windows": "yarn run eslint-components-windows && yarn run eslint-pages-windows",
    "stylelint-windows": "stylelint ./components/**/*.scss --config .stylelintrc.json && stylelint ./pages/**/*.scss --config .stylelintrc.json",
    "lint-windows": "yarn run eslint-windows && yarn run stylelint-windows"

File: pages/patient/poz.jsx [hunk 2]
```diff
@@ -0,0 +1,185 @@
+import Layout from '../../components/Layout/Layout';
+
+const CLASS_NAME = 'poz'
+
+const PozPage = () => (
```
Comment (Darex1991) [inline pages/patient/poz.jsx:5]: please rename it

File: pages/services/station.jsx [hunk 3]
```diff
@@ -0,0 +1,336 @@
+import Layout from '../../components/Layout/Layout'
+
+const CLASS_NAME = 'skp'
```
Comment (Darex1991) [inline pages/services/station.jsx:3]: please rename it

File: styles/_variables.scss [hunk 4]
```diff
@@ -6,19 +6,19 @@ $bgc_white: #ffffff;
...
 $youtube_red: #f44336;
 
-$font-size-xs: 12px;
-$font-size-sm: 14px;
-$font-size-base: 18px;
-$font-size-lg: 22px;
-$font-size-xl: 28px;
-$font-size-2xl: 48px;
-$font-size-3xl: 72px;
+$font-size-xs: 0.75rem;
```
Comment (Darex1991) [inline styles/_variables.scss:16]: please remove zero's

File: styles/about/videoStyles.scss [hunk 5]
```diff
@@ -0,0 +1,14 @@
+@import '../variables';
+
+.video {
+    display: flex;
+    flex-direction: column;
+    &__iframe {
+        max-width: 100%;
+        width: 100%;
```
Comment (Darex1991) [inline styles/about/videoStyles.scss:8]: alphabetic everywhere, please use linter

File: styles/globalStyles.scss [hunk 6]
```diff
@@ -38,19 +43,125 @@ body {
...
+        background-color: $accent;
+        color: $txt_color_light;
+        font-size: $font-size-sm;
+    }
+    td {
+        font-size: $font-size-sm;
+    }
+}
+
+.page__pdf-wrapper {
```
Comment (Darex1991) [inline styles/globalStyles.scss:88]: please nest into `.page` parent

File: styles/globalStyles.scss [hunk 7]
```diff
@@ -38,19 +43,125 @@ body {
...
+    border: solid 2px darken($bgc_light , 5);
+    &:hover, &:focus {
+        background: darken($bgc_light, 5);
+    }
+    .icon-wrapper {
+        width: 30px;
+        margin-bottom: 16px;
+        display: inline-flex;
+        height: 30px;
+        .pdf-icon {
```
Comment (Darex1991) [inline styles/globalStyles.scss:107]: please move this .pdf-icon into the parent. It does not need to be nested inside icon-wrapper.
Also, both should have a name from parent-like `page__pdf-icon`

File: styles/patient/nightCareStyles.scss [hunk 8]
```diff
@@ -0,0 +1,85 @@
...
+            width: auto;
+        }
+        &-text {
+            margin: 0;
+            &:nth-child(2) {
+                margin-top: 12px !important;
+            }
+        }
+        &-list-title {
+            font-weight: 700;
```
Comment (Darex1991) [inline styles/patient/nightCareStyles.scss:43]: please use variable

File: styles/services/stationStyles.scss [hunk 9]
```diff
@@ -0,0 +1,21 @@
+@import '../variables';
+
+.skp {
```
Comment (Darex1991) [inline styles/services/stationStyles.scss:3]: please change that name

Comment (snacksncode) [reply styles/services/stationStyles.scss:3]: to what exactly i didn't have any idea

Comment (Darex1991) [reply styles/services/stationStyles.scss:3]: to this `skp` it should be understandable for any developer

**Pull request merged**